vision_agent/tools/tools.py

```diff
@@ -395,6 +395,127 @@ def __call__(
         return rets
 
 
+class ZeroShotCounting(Tool):
+    r"""ZeroShotCounting is a tool that can count total number of instances of an object
+    present in an image belonging to same class without a text or visual prompt.
+
+    Example
+    -------
+        >>> import vision_agent as va
+        >>> zshot_count = va.tools.ZeroShotCounting()
+        >>> zshot_count("image1.jpg")
+        {'count': 45}
+    """
+
+    name = "zero_shot_counting_"
+    description = """'zero_shot_counting_' is a tool that can count total number of instances of an object present in an image belonging to the same class without a text or visual prompt.
+    It returns the total count of the objects."""
```

Comment:
With the newline this might mess up the prompt for tool choice, because right now each tool is on one single line

Reply:
OK will make it single line. It was going way beyond 80, so thought would wrap it. If its going to create an issue with the tool choice then lets make it single line